Assignments/palindrome/PalindromeTest.java

```diff
@@ -0,0 +1,33 @@
+package palindrome;
+
+import java.util.Stack;
+
+import java.util.Scanner;
+
+public class PalindromeTest 
+{
+	public static void main(String[] args)
+	{
+		System.out.println("Enter any string:");
+		Scanner in=new Scanner(System.in);
+		String inputString = in.nextLine();
+		Stack<Character> stack = new Stack<Character>();
+		
+		for(int i=0;i<inputString.length();i++)
+		{
+			stack.push(inputString.charAt(i));
+    	}
+	
+    	String reverseString = "";
```

Comment:
Variable should be declared at top of the class.

Reply:
done